integration-test-groups/foundation/bean/src/main/java/org/apache/camel/quarkus/component/bean/bind/BindToRegistryOnMethodBean.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.bean.bind;
+
+import org.apache.camel.BindToRegistry;
+
+public class BindToRegistryOnMethodBean {
+    @BindToRegistry
```

Comment:
So, `BindToRegistryBean` class is included via annotated field and method ? Would it be better for test coverage to use distinct classes ala `BindToRegistryBean1` `BindToRegistryBean2` ? With better names of course :)

Reply:
> use distinct classes

I could, but the bean type doesn't matter so much (IMO).

I think what we want to verify is whether a bean was bound to registry with the expected name and is resolvable in a Camel route or via a manual lookup.